packages/react/src/PricingOptions/PricingOptions.stories.tsx

```diff
@@ -0,0 +1,133 @@
+      paddingBlockEnd="spacious"
+      style={{
+        ['--brand-color-accent-primary' as string]: 'var(--base-color-scale-purple-4)',
+        minHeight: '100vh',
+      }}
+    >
+      <Grid>
+        <Grid.Column span={12}>
+          <PricingOptions>
+            <PricingOptions.Item>
+              <PricingOptions.Heading>Copilot Individual</PricingOptions.Heading>
+              <PricingOptions.Description>
+                Code completions, Chat, and more for indie developers and freelancers.
+              </PricingOptions.Description>
+              <PricingOptions.Price currencyCode="USD" currencySymbol="$" trailingText="per month / $100 per year">
+                10
+              </PricingOptions.Price>
+              <PricingOptions.FeatureList>
+                <PricingOptions.FeatureListHeading>Chat</PricingOptions.FeatureListHeading>
+                <PricingOptions.FeatureListItem>
+                  Unlimited messages, interactions, and history
+                </PricingOptions.FeatureListItem>
+                <PricingOptions.FeatureListItem>
+                  Context-aware coding support and explanations
+                </PricingOptions.FeatureListItem>
+                <PricingOptions.FeatureListItem variant="excluded">
+                  Debugging and security remediation assistance
+                </PricingOptions.FeatureListItem>
+                <PricingOptions.FeatureListItem variant="excluded">
+                  Repository-based semantic search
+                </PricingOptions.FeatureListItem>
+                <PricingOptions.FeatureListItem variant="excluded">
+                  Access your knowledge base
+                </PricingOptions.FeatureListItem>
+                <PricingOptions.FeatureListHeading>Code completion</PricingOptions.FeatureListHeading>
+                <PricingOptions.FeatureListItem>Code suggestions as you type</PricingOptions.FeatureListItem>
+                <PricingOptions.FeatureListItem>Comments to code</PricingOptions.FeatureListItem>
+                <PricingOptions.FeatureListItem variant="excluded">
+                  Fine-tuned models (coming soon)
+                </PricingOptions.FeatureListItem>
+              </PricingOptions.FeatureList>
+
+              <PricingOptions.Footnote>
+                Free for verified students, teachers, and maintainers of popular open source projects.
+              </PricingOptions.Footnote>
+
+              <PricingOptions.PrimaryAction href="/buy">Start a free trial</PricingOptions.PrimaryAction>
+            </PricingOptions.Item>
+
+            <PricingOptions.Item>
+              <PricingOptions.Label>Recommended</PricingOptions.Label>
+              <PricingOptions.Heading>Copilot Business</PricingOptions.Heading>
+              <PricingOptions.Description>Copilot personalized to your organization.</PricingOptions.Description>
+              <PricingOptions.Price currencyCode="USD" currencySymbol="$" trailingText="per user / month">
```

Comment:
I think I saw the currency code and symbol defaulting to USD/$ in the component so do we need to add the props here? Maybe use GBP/$ or something different here instead?